controllers/articles.js

```diff
@@ -116,7 +125,7 @@ const articles = {
           body: updateThisArticle.body,
           slug: updateThisArticle.slug,
           ratings: updateThisArticle.ratings,
-          tagList: updateThisArticle.tagList,
+          tagList: updateThisArticle.tagList
```

Comment:
For code refactoring (particularly on the object), it is better if you leave those comas there.